CHANGELOG.md

```diff
@@ -7,16 +7,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.1.0] 2024-01-04
+
 ### Created
 
 - Created a campaign hero component and other child components for the featured item data. (DR-2464)
 
 ### Update
+
 - Update version of NYPL Reservoir to 2.0.1
 - Updated image in Hero Component to change on reload. (DR-2565)
+- Updated number in Hero Component to use a live number in Repo API. (DR-2543)
+- Updated featured item link in Hero Component to change on reload using the featured item imageID. (DR-2547)
+- Updated travis deployer keys after rotation (DR-2722)
 
 ### Added
+
 - Added spider block script (DR-2518)
 - Created Dockerfile and docker-compose.yml files. (DR-2446)
 - Created Next.js 13 app using NYPL Reservoir and TS 4.9.5. (DR-2444)
 - Set up travis script to build and deploy to QA (DR-2448)
+- Added global footer (DR-2591)
```

Comment:
I haven't been adding to the changelog... should I add in all my tickets before this goes to prod

Reply:
yes lol you can add them to this release branch or make a new branch and make a PR to merge into main, I don't have a preference